api_tests/e2e/rfc003/btc_eth/refund.ts

```diff
@@ -1,135 +1,44 @@
-
-(async function() {
-    const alice = new Actor("alice", {
-        ledgerConfig: global.ledgerConfigs,
-        addressForIncomingBitcoinPayments:
-            "bcrt1qs2aderg3whgu0m8uadn6dwxjf7j3wx97kk2qqtrum89pmfcxknhsf89pj0",
+import { Actor } from "../../../lib_sdk/actors/actor";
+import { AssetKind } from "../../../lib_sdk/asset";
+import { createActors } from "../../../lib_sdk/create_actors";
+
+setTimeout(function() {
+    let alice: Actor;
+    let bob: Actor;
+
+    beforeEach(async function() {
+        this.timeout(20000);
+        const actors = await createActors("e2e-rfc003-btc-eth-refund.log");
+        alice = actors.alice;
+        bob = actors.bob;
     });
-    const bob = new Actor("bob", {
-        ledgerConfig: global.ledgerConfigs,
-    });
-
-    const alphaAssetQuantity = 100000000;
-    const betaAssetQuantity = ethers.utils.parseEther("10");
-    const maxFeeInSatoshi = 50000;
 
-    const alphaExpiry = Math.round(Date.now() / 1000) + 13;
-    const betaExpiry = Math.round(Date.now() / 1000) + 8;
-
-    await bitcoin.ensureFunding();
-    await bob.wallet.eth().fund("11");
-    await alice.wallet.eth().fund("0.1");
-    await alice.wallet.btc().fund(10);
-
-    const bobInitialWei = await bob.wallet.eth().ethBalance();
-
-    const swapRequest: SwapRequest = {
-        alpha_ledger: {
-            name: "bitcoin",
-            network: "regtest",
-        },
-        beta_ledger: {
-            name: "ethereum",
-            chain_id: 17,
-        },
-        alpha_asset: {
-            name: "bitcoin",
-            quantity: alphaAssetQuantity.toString(),
-        },
-        beta_asset: {
-            name: "ether",
-            quantity: betaAssetQuantity.toString(),
-        },
-        beta_ledger_redeem_identity: alice.wallet.eth().address(),
-        alpha_expiry: alphaExpiry,
-        beta_expiry: betaExpiry,
-        peer: await bob.peerId(),
-    };
+    afterEach(() => {
+        if (alice) {
+            alice.stop();
+        }
+        if (bob) {
+            bob.stop();
+        }
+    });
 
-    const steps: Step[] = [
-        {
-            actor: bob,
-            action: ActionKind.Accept,
-            waitUntil: state => state.communication.status === "ACCEPTED",
-        },
-        {
-            actor: alice,
-            action: ActionKind.Fund,
-            waitUntil: state => state.alpha_ledger.status === "FUNDED",
-        },
-        {
-            actor: bob,
-            action: ActionKind.Fund,
-            waitUntil: state =>
-                state.alpha_ledger.status === "FUNDED" &&
-                state.beta_ledger.status === "FUNDED",
-            test: {
-                description: "Should have less beta asset after the funding",
-                callback: async () => {
-                    const bobWeiBalanceAfter = await bob.wallet
-                        .eth()
-                        .ethBalance();
+    describe("Bob can refund and then Alice refunds", function() {
```

Comment:
```suggestion
    describe("Bob refunds then Alice refunds", function() {
```
I find the inconsistent use of `can` confusing.